### ISSUE TITLE
[Research] OUI Compliance audit of the `input_control_vis` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4083]
Tracking issue for OUI compliance audit of the `input_control_vis` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by PreetiToppo** [comment]
Hi @BSFishy, I'm new to open source and would like to take on this OUI compliance audit for the input_control_vis plugin. I've read through #3945 to understand the process. Please assign this to me if it's still available. Thanks!

**Comment by BSFishy** [comment]
Hey @PreetiToppo I'm no longer a maintainer of this project. I would recommend getting the attention of someone on https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/MAINTAINERS.md to take up this work
